### ISSUE TITLE
Chapter 3 Comments

### ISSUE THREAD

**Issue by anniecollins** [RohanAlexander/telling_stories#5]
- Same comment as chapter 2 about writing some tidyverse packages separately
- Given the formatting in the “Key Functions” section, I changed the operators in the body of the text to have code formatting instead of being in quotations.
- What is the logic behind how “tidyverse” is stylized in this chapter? I notice you switch back and forth between “the tidyverse” and “the `tidyverse`" (for example, the paragraph on line 96). It seems to be inconsistent and I can’t tell if there’s a pattern that I’m missing.
- Line 98: I felt like this paragraph was a bit unclear so I added a bit to the final sentence to clarify, feel free to delete.
- Approx. line 253: It may be worth mentioning the tidyverse/magrittr pipe somewhere in here so that folks are able to recognize both `%>%` and `|>` if they’re learning R for the first time.
- Line 273: Worth mentioning `readr` as a package for importing data?
- Approx line 487: Might be worth mentioning the `n()` function prior to this code chunk, or explaining the code below in plain english.
- Between line 534 and 545 you switched from sorting by ‘birthDate’ to ‘birthYear’. I’m not sure what the logic is behind this and I feel like it could be confusing to the reader.
- Line 723: What does “by house” mean here? Might be good to explain how this connects to the ‘member’ variable
- `class()` section: The stylization of the class names (’factor’, ‘numeric’, etc.) was inconsistent throughout this section. Was there a reason for the names sometimes being in quotations and other times not? I went through and updated the formatting to be in quotations consistently but if this is unnecessary then my apologies.
- `class()` section: It may be worth mentioning somewhere in this section that the class of a vector is whatever the class of its contents is. In Python and other languages, a similar data structure to a vector is a ‘list’. A ‘list’ is a class of its own, and the objects in a ‘list’ have their own classes (for example, [’a’, 1] is an object of class ‘list’ with entries of class ‘string’ and ‘int’). It may be counterintuitive for people who know other programming languages to see that a vector is not its own class. However, depending on your intended audience, this may not be a concern.
- Line 1013: What is the point of these two code chunks? Am I missing something? I think the example with simulated_data suffices.
- Line 1290: Define wide/long data
- Exercises: Question 13 seems irrelevant, `exp()` is not covered in this section and this does not test any knowledge of `mutate()`

**Comment by RohanAlexander** [comment]
> * Same comment as chapter 2 about writing some tidyverse packages separately

Have changed the 'key libraries' bit here to have a sub-list. Will need to fix this within the text when I notice them. 

> * Given the formatting in the “Key Functions” section, I changed the operators in the body of the text to have code formatting instead of being in quotations.

Okay, thank you.

> * What is the logic behind how “tidyverse” is stylized in this chapter? I notice you switch back and forth between “the tidyverse” and “the `tidyverse`" (for example, the paragraph on line 96). It seems to be inconsistent and I can’t tell if there’s a pattern that I’m missing.

I think I need to try to use 'the tidyverse'. I'll need to change this on another read-through.

> * Line 98: I felt like this paragraph was a bit unclear so I added a bit to the final sentence to clarify, feel free to delete.

Good changes. Thank you.

> * Approx. line 253: It may be worth mentioning the tidyverse/magrittr pipe somewhere in here so that folks are able to recognize both `%>%` and `|>` if they’re learning R for the first time.

Good point. Have added a paragraph. Would be confusing.

> * Line 273: Worth mentioning `readr` as a package for importing data?

Have added

> * Approx line 487: Might be worth mentioning the `n()` function prior to this code chunk, or explaining the code below in plain English.

Have added a brief explanation in words.

> * Between line 534 and 545 you switched from sorting by ‘birthDate’ to ‘birthYear’. I’m not sure what the logic is behind this and I feel like it could be confusing to the reader.

Have changed them all to 'birthYear'

> * Line 723: What does “by house” mean here? Might be good to explain how this connects to the ‘member’ variable

Have changed the language.

> * `class()` section: The stylization of the class names (’factor’, ‘numeric’, etc.) was inconsistent throughout this section. Was there a reason for the names sometimes being in quotations and other times not? I went through and updated the formatting to be in quotations consistently but if this is unnecessary then my apologies.

Thank you.

> * `class()` section: It may be worth mentioning somewhere in this section that the class of a vector is whatever the class of its contents is. In Python and other languages, a similar data structure to a vector is a ‘list’. A ‘list’ is a class of its own, and the objects in a ‘list’ have their own classes (for example, [’a’, 1] is an object of class ‘list’ with entries of class ‘string’ and ‘int’). It may be counterintuitive for people who know other programming languages to see that a vector is not its own class. However, depending on your intended audience, this may not be a concern.

Have basically just added this verbatim. Will cut you a percentage of the royalties. 

> * Line 1013: What is the point of these two code chunks? Am I missing something? I think the example with simulated_data suffices.

Looks like you commented them out already.

> * Line 1290: Define wide/long data

Have added a definition and example.

> * Exercises: Question 13 seems irrelevant, `exp()` is not covered in this section and this does not test any knowledge of `mutate()`

Good pick up. Have changed the the variation to be in `mutate` not in `exp`.
